src/lib/database/database.config.ts

```diff
@@ -20,7 +20,8 @@ export const config: ConnectionOptions = {
 		migrationsDir: 'src/lib/database/migrations',
 		subscribersDir: 'src/lib/database/subscribers'
 	},
-	namingStrategy: new SnakeNamingStrategy()
+	namingStrategy: new SnakeNamingStrategy(),
+	logging: true
```

Comment:
Make this a configuration key in config.ts, please.

Reply:
I'll set it to DEV